.mkdocs.yml

```diff
@@ -0,0 +1,37 @@
+site_name: Dimsum Documentation
```

Comment:
I've been using "Dimsum" personally since the logo doesn't have a space. While the logo also doesn't have a capital, I think it should be capitalized as a proper noun. We can discuss at the next meeting and make it consistent after that.